### DISCUSSION TITLE
Adding support for Association of Computational Linguistics publications?

### DISCUSSION THREAD

**Discussion by eahogue** [GjjvdBurg/paper2remarkable#92] [Ideas]
Hi,

First off, love, love, love this tool. Thank you for making it. 

It's great that you have support for so many academic paper repositories. I wonder if you would consider adding one more:

https://www.aclweb.org/portal/

It's pretty much the default for papers in computational linguistics/nlp. It would be nice to have support for it.

But either way, thanks!

Alan

**Comment by GjjvdBurg** [comment]
Hi Alan,

Thanks for the kind words, glad to hear you find the script useful!

ACL is indeed a great source to add, since it hosts so many different conferences through a single interface. I've added this in #93 and it should be in the updated package soon.

Kind regards,

Gertjan
